### PR TITLE
Update openshift-cincinnati-master.yaml

### PR DESCRIPTION
**Pull request by shiywang**
Wire YAML lint test up into the prow job
follow-up work of https://github.com/openshift/cincinnati/pull/85/files

### CODE REVIEW

**Comment by shiywang** [comment]
do we want to gather all the test script into one command like sanity_test or sth?

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml [hunk 1]
```diff
@@ -28,3 +28,7 @@ tests:
   commands: dist/pr_check.sh
```
Comment (steveeJ) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml]: let's rename this script to `cargo_test`

Comment (shiywang) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml]: done, also https://github.com/openshift/cincinnati/pull/110

**Pull request merged**